index.html

```diff
@@ -20,54 +20,68 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <aside class="menu-bar hidden-menu">
```

Comment:
Kindly fix the horizontal overflowing issue. As you may know these kinds of overflows are belonging to the pre-responsive era. It is expected that modern web pages are fully responsive because of the high volume of mobile users in the Internet. 👍
![Screen Recording 2023-01-30 at 9 36 19 PM](https://user-images.githubusercontent.com/25887253/215569560-251d1e0b-5bf9-44ed-a832-c012d27da26c.gif)